provisioner/ibm-s3fs-provisioner_test.go

```diff
@@ -516,6 +539,38 @@ func Test_Provision_AutoDeleteBucketWithoutAutoCreateBucket(t *testing.T) {
 	}
 }
 
+func Test_Provision_SetDefault_AutoCreateBucket_AutoDeleteBucket_BucketName(t *testing.T) {
+	p := getProvisioner()
+	v := getVolumeOptions()
+	v.PVC.Annotations[annotationAutoCreateBucket] = ""
+	v.PVC.Annotations[annotationAutoDeleteBucket] = ""
```

Comment:
Do we really need to specify annotation (only for line 545 and 546)if its empty? Not specifying annotation wont be treated as empty?